src/config_handling.py

```diff
@@ -43,20 +43,27 @@
     "utility_interval",
```

Comment:
A comment about how "master" and "worker" are deprecated and will eventually be removed should be added to the configuration handler.

Reply:
Agreed, will do.

Reply:
Done.